### PR TITLE
[M1133] User metrics

### PR DESCRIPTION
**Pull request by gridcell**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new Django admin interface for managing user metrics.
	- Added functionality for executing daily tasks, including database backups and metrics updates.
	- Created a new model to capture user-related metrics, enhancing data tracking capabilities.

- **Bug Fixes**
	- Improved handling of log events and aggregation of metrics, including counting image uploads.

- **Documentation**
	- Updated help texts for new management commands to clarify their purposes.

### CODE REVIEW

**Comment by gridcell** [comment]
@coderabbitai review

**Comment by gridcell** [comment]
[make user_metrics an updated table rather than a csv export](https://trello.com/c/6zPmCyqI/1133-make-usermetrics-an-updated-table-rather-than-a-csv-export)

File: src/tools/management/commands/update_metrics.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
```
Comment (saanobhaai) [inline src/tools/management/commands/update_metrics.py:1]: We should deal with output_user_metrics.py and the bucket it writes to. Should we delete both? Or consolidate with this command as an option?

File: src/tools/management/commands/update_metrics.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (saanobhaai) [inline src/tools/management/commands/update_metrics.py:2]: Doing just one day at a time makes sense, but we still need the table to include the aggregations of all log events from the beginning (the way output_user_metrics does). Could be a one-time data migration?

**Review by saanobhaai — CHANGES_REQUESTED** [review]

File: src/tools/management/commands/update_metrics.py (hunk 1 above)
Comment (gridcell) [reply src/tools/management/commands/update_metrics.py:1]: Removed command

File: src/tools/management/commands/update_metrics.py (hunk 2 above)
Comment (gridcell) [reply src/tools/management/commands/update_metrics.py:2]: Manually run on server with a defined date range.

**Review by saanobhaai — APPROVED** [review]

**Pull request merged**